k8s/nginx/service.yml

```
@@ -0,0 +1,13 @@
+kind: Service
+metadata:
+  name: nginx-service
+spec:
+  type: LoadBalancer
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 80
+  selector:
+    app: nginx-deployment
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)